src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -400,6 +427,12 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
           ? internalSelectedInputAsset.value?.uniqueId !== extendedAsset?.uniqueId
           : internalSelectedOutputAsset.value?.uniqueId !== extendedAsset?.uniqueId;
 
+      analyticsV2.track(analyticsV2.event.swapsSelectedAsset, {
```

Comment:
Probably a minor difference but I'd put this at the end of the setAsset function